src/pages/profiles/forms/ProfileFormMenu.tsx

```diff
@@ -87,7 +89,9 @@ const ProfileFormMenu: FC<Props> = ({
               />
               <MenuItem label={GPU_DEVICES} {...menuItemProps} />
               <MenuItem label={PROXY_DEVICES} {...menuItemProps} />
-              <MenuItem label={OTHER_DEVICES} {...menuItemProps} />
+              {!hasMetadataConfiguration && (
```

Comment:
```suggestion
              {hasMetadataConfiguration && (
```
I think this needs to be inversed.

Reply:
oops, that's my bad!